src/pages/deployment/config-reference.mdx

```diff
@@ -77,6 +79,12 @@ In the following properties, `<applicationID>` is a placeholder for the name of
 | `apps.sessionConfig.sessionTimeout` | Amount of time a session can go unused before it is no longer valid | `30m` |
 | `apps.sessionConfig.transactionTimeout` | Maximum time allowed for transactions started on this server to complete. Any such transactions that do not complete before this timeout occurs are rolled back. | `3m` |
 | `apps.systemUser.credsSecretName` | Name of a pre-created secret containing credentials of the system user (see [Handling Secrets](/deployment/secrets) topic for expected secret content) | `SYSTEM` |
+| `apps.tuningDefaults.coreThreads` | Minimum number of threads to associate with the [executor](https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.liberty.autogen.base.doc/ae/rwlp_config_executor.html) | `4` |
+| `apps.tuningDefaults.maxThreads` | Maximum number of threads that can be associated with the [executor](https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.liberty.autogen.base.doc/ae/rwlp_config_executor.html) | `4` |
+| `apps.tuningDefaults.maxPoolSize` | Maximum number of physical connections for a [pool](https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.liberty.autogen.base.doc/ae/rwlp_config_connectionManager.html) | `8` |
+| `apps.tuningDefaults.numConnectionsPerThreadLocal` | Number of connections to be cached for each thread | `2` |
```

Comment:
missing link to external documentation. should ling to https://www.ibm.com/docs/en/was-liberty/base?topic=configuration-connectionmanager